RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h

```diff
@@ -30,7 +30,8 @@ namespace gpuClustering {
     __shared__ uint16_t newclusId[maxNumClustersPerModules];
 
     constexpr int startBPIX2 = isPhase2 ? phase2PixelTopology::layerStart[1] : phase1PixelTopology::layerStart[1];
-    constexpr int nMaxModules = isPhase2 ? phase2PixelTopology::numberOfModules : phase1PixelTopology::numberOfModules;
+    [[maybe_unused]] constexpr int nMaxModules =
+        isPhase2 ? phase2PixelTopology::numberOfModules : phase1PixelTopology::numberOfModules;
```

Comment:
I suppose these are to silence the compilation warnings.
Out of curiosity, using them I an assertion one line below doesn't count as "use"?

Reply:
Correct.

No, because on GPUs we disable assertions, so the content of `assert(...)` is never seen by the compiler.

Reply:
Thanks